### ISSUE TITLE
False-positive in TYPE_ALIAS inspection

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#317]
`typealias RulesConfigReaderBase = JsonResourceConfigReader<List<RulesConfig>>`
causes
```
[INFO]    [ktlint] ...\diktat-common\src\main\kotlin\org\cqfn\diktat\common\config\rules\RulesConfigReader.kt:46:35: [TYPE_ALIAS] variable's type is too complex and should be replaced with typealias: too long type reference (cannot be auto-corrected)
```

Value assigned to a typealias shouldn't be checked on complexity.
